lib/src/editor/toolbar/mobile/toolbar_items/heading_mobile_toolbar_item.dart

```diff
@@ -47,45 +47,54 @@ class _HeadingMenuState extends State<_HeadingMenu> {
   @override
   Widget build(BuildContext context) {
     final style = MobileToolbarStyle.of(context);
+    final size = MediaQuery.of(context).size;
```

Comment:
```suggestion
    final size = MediaQuery.size;
```
I believe this should also be possible.

Reply:
`MediaQuery.size`? How to get the size without contenxt?

Reply:
It's my bad. I wanted to write `sizeOf(context)`.